starter/src/styles/_theme.scss

```diff
@@ -1,7 +1 @@
-@import "~@angular/material/theming";
```

Comment:
@christophercr, is it possible this one was in the wrong spot and should have been imported after `@import "variables";`?